src/MonitoringFunctions/Properties/serviceDependencies.dotnet-install-daemon-functions - Zip Deploy.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "secrets1": {
+      "resourceId": "/subscriptions/[parameters('subscriptionId')]/resourceGroups/[parameters('resourceGroupName')]/providers/Microsoft.KeyVault/vaults/Dotnet-Install-Keys",
+      "type": "secrets.keyVault",
+      "connectionId": "monitoring-functions-push-connection-string",
+      "secretStore": null
+    },
+    "storage1": {
+      "resourceId": "/subscriptions/[parameters('subscriptionId')]/resourceGroups/[parameters('resourceGroupName')]/providers/Microsoft.Storage/storageAccounts/monitoringfuncsstorage",
+      "type": "storage.azure",
+      "connectionId": "AzureWebJobsStorage",
+      "secretStore": "AzureKeyVault"
+    }
+  }
+}
```

Comment:
white space

Reply:
Must be placed by github. There is no whitespace in the file.